### PR TITLE
Create plugin-graphql-server.yml

### PR DESCRIPTION
**Pull request by halkeye**
# Description

https://issues.jenkins-ci.org/browse/HOSTING-784
https://github.com/jenkinsci/graphql-server-plugin
No previous maintainers

### Always

- [X] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [X] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [X] Make sure the file is created in `permissions/` directory
- [X] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [X] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [X] Check that the file is named `plugin-${artifactId}.yml` for plugins

### CODE REVIEW

File: permissions/plugin-graphql-server.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "graphql-plugin"
+github: "jenkinsci/graphql-plugin"
+paths:
+- "org/jenkins-ci/plugins/graphql-plugin"
```
Comment (slide) [inline permissions/plugin-graphql-server.yml]: This path is incorrect, it should match the groupId/artifactId from your pom.xml (https://github.com/jenkinsci/graphql-server-plugin/blob/master/pom.xml#L9). So, you would need "io/jenkins/plugins/graphql-server"

Comment (halkeye) [reply permissions/plugin-graphql-server.yml]: thanks, skimmed it too fast.

**Comment by slide** [comment]
Looks like the name and GitHub Peyton are incorrect too. The plugin repo is  graphql-server-plugin, and the name is graphql-server.

**Comment by halkeye** [comment]
thanks for catching that too
i fixed one earlier, I think they line up now.

File: permissions/plugin-graphql-server.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "graphql-plugin"
```
Comment (slide) [inline permissions/plugin-graphql-server.yml]: I think the name still needs to be graphql-server (plugin name in pom?)

**Comment by halkeye** [comment]
@slide your absolutely right. I've been doing most of the fixes to the file on mobile and not giving it enough attention.

I hope that's the last fix I do on mobile.

I'll be back on a computer in a couple hours and look more carefully.

**Pull request merged**